README.md

```diff
@@ -1,8 +1,304 @@
+case a summary from the meeting will be posted back to the RFC pull request.
+
+A sub-team makes final decisions about RFCs after the benefits and drawbacks
+are well understood. These decisions can be made at any time, but the sub-team
+will regularly issue decisions. When a decision is made, the RFC pull request
+will either be merged or closed. In either case, if the reasoning is not clear
+from the discussion in thread, the sub-team will add a comment describing the
+rationale for the decision. In the case of the RFC being closed, the tag
+"RFC: Declined" will be added to the tracking issue. If it is merged, it will
+receive the tag "RFC: Accepted".
+
+### Implementing an RFC
+[Implementing an RFC]: #implementing-an-rfc
+
+Some accepted RFCs represent vital features that need to be implemented right
+away. Other accepted RFCs can represent features that can wait until some
+arbitrary developer feels like doing the work. Every accepted RFC has an
+associated issue tracking its implementation in the Amethyst repository; thus that
+associated issue can be assigned a priority via the triage process that the
+team uses for all issues in the Amethyst repository.
+
+The author of an RFC is not obligated to implement it. Of course, the RFC
+author (like any other developer) is welcome to post an implementation for
+review after the RFC has been accepted. A prototype or other supporting
+materials (data flow diagrams, architecture diagrams, aggregations of
+posts from the community) can all help demonstrate the need and the
+priority of the RFC.
+
+If you are interested in working on the implementation for an approved RFC, but
+cannot determine if someone else is already working on it, feel free to ask
+(e.g. by leaving a comment on the associated issue).
+
+Once an RFC has been implemented, the tracking issue will be closed and given
+the label "RFC: Complete".
+
+### RFC Postponement
+[RFC Postponement]: #rfc-postponement
+
+Some RFC pull requests are tagged with the "RFC: Postponed" label when they are
+closed (as part of the rejection process). An RFC closed with "RFC: Postponed" is
+marked as such because we want neither to think about evaluating the proposal
+nor about implementing the described feature until some time in the future, and
+we believe that we can afford to wait until then to do so. Postponed pull
+requests may be re-opened when the time is right. We don't have any formal
+process for that, you should ask members of the relevant sub-team.
+
+Usually an RFC pull request marked as "RFC: Postponed" has already passed an
+informal first round of evaluation, namely the round of "do we think we would
+ever possibly consider making this change, as outlined in the RFC pull request,
+or some semi-obvious variation of it." (When the answer to the latter question
+is "no", then the appropriate response is to close the RFC, not postpone it.)
+
+### Help! This is all too informal
```

Comment:
I never really liked this section in the rust-lang RFC process.  It's excessively formal in my opinion.  That doesn't mean it's bad, but there is a lot more ceremony involved in this process compared to what we've done previously.

Reply:
@Xaeroxe Do you think it should be removed entirely? Or do you have a rewording in mind?

Reply:
I'd remove it entirely personally

Reply:
Are you ok with leaving it in for this round, and we can do another round to focus on that section after it is merged? I feel like removing an entire section would require another full round of approvals...

Reply:
Sure